psycop/common/model_training_v2/hyperparameter_suggester/suggesters/test_suggesters.py

```diff
@@ -47,10 +48,18 @@ def suggester_tester(suggester: Suggester) -> TestSuggestion:
 def test_logistic_regression_suggester():
     result = suggester_tester(
         suggester=LogisticRegressionSuggester(
-            C=(0,1,False),
-            l1_ratio=(0,1,False),
+            C_low=0.1,
+            C_high=1,
+            C_log=False,
```

Comment:
Had to change the interface because confection cannot resolve tuples from the .cfg